### PR TITLE
[Feature] Added aggregations without type dimension

### PR DESCRIPTION
**Pull request by nup9151f**
## Summary

Added aggregations over transactions without `type` dimension. Renamed aggregations to more appropriate names.

## Description

Aggregation have been renamed to use the following pattern: **TransactionDimension1Dimension2...Stat**.

Meaning if we want aggregation by `Type` , `User` and `App` we should query aggregation `TransactionTypeUserAppStat`

New aggregations where added that do not have the `type` dimension. This change facilitates calculations given that we do not have to loop over all different types to get totals.

## Related Issue/Bounty

OF-283 - Difficult to get transaction totals

## Type of Change

Please check the boxes that apply to your pull request:

- [X] Breaking change (fix or feature that would cause existing functionality to change)

All previous queries need to be updated because aggregations have changed names.

### CODE REVIEW

**Review by george-openformat — APPROVED** [review]
Fantastic the naming convention is great!

**Comment by george-openformat** [comment]
hey @tinypell3ts we are blocked on merging this as **code owner review is required**

**Review by tinypell3ts — APPROVED** [review]

**Pull request merged**
